pkgs/top-level/release.nix

```diff
@@ -11,7 +11,7 @@
 { nixpkgs ? { outPath = (import ../../lib).cleanSource ../..; revCount = 1234; shortRev = "abcdef"; revision = "0000000000000000000000000000000000000000"; }
 , officialRelease ? false
   # The platforms for which we build Nixpkgs.
-, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" ]
+, supportedSystems ? [ "x86_64-linux" "x86_64-darwin" "aarch64-linux" "aarch64-darwin" ]
```

Comment:
Is this right? will this not block channels?

Reply:
it will only block for failing constituents

Reply:
Nevermind. We have an explicit list of jobs + arch here since a while..

Reply:
It will, about as much as `x86_64-darwin`, but I'd hope it works out OK.

Reply:
IIUC, this will just add the aarch64-darwin packages to the jobset. Channel blocking is done by adding certain jobs. We should probably have a follow-up PR which at least has the fundamental packages listed so we don't ship completely broken aarch64-darwin updates. But that can be done in another PR.

Reply:
In particular note that NixOS channels won't be affected, just as they're not affected by `x86_64-darwin`.

Reply:
A more interesting question (for later than this PR) is what the `nixpkgs-21.05-darwin` channel should include...

Reply:
> A more interesting question (for later than this PR) is what the `nixpkgs-21.05-darwin` channel should include...

probably both x86_64 and aarch64